portality/scripts/remove_duplicate_articles.py

```diff
@@ -46,15 +61,19 @@
         if duplicates:
```

Comment:
This line: XWalk().get_duplicate(a, all_duplicates=True)

all_duplicates is no longer part of the signature for that method, you want XWalk().get_duplicates() instead